iaso/api/user_roles.py

```diff
@@ -98,8 +95,19 @@ def update(self, user_role, validated_data):
 
         group.save()
         user_role.save()
+        user_role.editable_org_unit_types.set(editable_org_unit_types)
         return user_role
 
+    def validate_editable_org_unit_types(self, editable_org_unit_types):
+        account = self.context.get("request").user.iaso_profile.account
+        project_org_unit_types = set(Project.objects.get(account=account).unit_types.values_list("id", flat=True))
```

Comment:
This can lead to this error : 
![Screenshot 2024-10-21 at 12 49 20](https://github.com/user-attachments/assets/55dd92f5-5855-4b89-8b0d-426f6bcbd7bf)

Reply:
Should be fixed in https://github.com/BLSQ/iaso/pull/1722/commits/46c55e1a3c152d2cfbda2f96b874253e8f9ea9bc